pkgs/tools/security/cve-bin-tool/default.nix

```diff
@@ -100,7 +56,11 @@ buildPythonApplication rec {
 
   propagatedBuildInputs = [
```

Comment:
I somewhat doubt that many of these have to be here, only python deps have to be here.

Reply:
I did some testing and since pip only has to be in the path during runtime it should be in `propagatedBuildInputs`.
In its current form it isn't actually used.